### PR TITLE
Enhancement/utils refactor

### PR DESCRIPTION
**Pull request by jzeltman**
Issue: Util inclusion was getting messy

## What I did
I've added an cli/src/utils/index file to include all of the utilities, and updated the references to use this file.

## How to test
Review imports, check they're as expected. 

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: cli/src/index.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
 const chalk = require('chalk');
 const logger1 = require('./utils/logger');
-//const { checkVersion } = require('./utils/versionCheck');
+// const { checkVersion } = require('./utils');
```
Comment (CodeByAlex) [inline cli/src/index.ts]: can this be removed?

File: cli/src/index.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
+/* eslint-disable global-require */
 module.exports = () => {
   const args = process.argv.slice(2);
   const cmd = args[0];
 
   // We can't check the version until this package is actually published on NPM.
-  //checkVersion();
+  // checkVersion();
```
Comment (CodeByAlex) [inline cli/src/index.ts]: can this be removed?

File: cli/src/index.ts (hunk 1 above)
Comment (jzeltman) [reply cli/src/index.ts]: Let's keep this in here for now. I think we can uncomment when we publish

File: cli/src/index.ts (hunk 2 above)
Comment (jzeltman) [reply cli/src/index.ts]: Let's keep this in here for now. I think we can uncomment when we publish

**Pull request merged**